sass/layout/hero.sass

```diff
@@ -17,7 +17,7 @@
     &.is-#{$name}
       background-color: $color
       color: $color-invert
-      a:not(.button),
+      a:not(.button, .dropdown, .tag),
```

Comment:
changing this to `a:not(.button):not(.dropdown):not(.dropdown-item):not(.tag),` produces the desired results.